clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -75,6 +77,33 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
     def currentTargetPoolUrls = managedInstanceGroup.getTargetPools()
     def newTargetPoolUrls = []
 
+
+    if (credentials.consulConfig?.enabled) {
+      task.updateStatus phaseName, "$presentParticipling server group in Consul..."
+      currentTargetPoolUrls.each { targetPoolUrl ->
+        def targetPoolLocalName = GCEUtil.getLocalName(targetPoolUrl)
+
+        task.updateStatus phaseName, "$presentParticipling instances with target pool $targetPoolLocalName in Consul..."
+
+        def targetPool = compute.targetPools().get(project, region, targetPoolLocalName).execute()
+        def instanceNames = targetPool.getInstances()?.collect { url ->
+          GCEUtil.getLocalName(url)
+        } ?: []
+
+        instanceNames.each { name ->
+          try {
+            EnableDisableConsulInstance.operate(credentials.consulConfig,
+                                                name,
+                                                disable
+                                                 ? EnableDisableConsulInstance.State.disable
```

Comment:
Even though the indentation is off here, the formatting feels like a small victory.